### DISCUSSION TITLE
Rerunning a task under a cursor in task list

### DISCUSSION THREAD

**Discussion by Qizot** [stevearc/overseer.nvim#121] [Q&A] [answered]
Just like in the title, is it possible to somehow rerun a task that is currently selected in task list sidebar? And if not then what is necessary to implement that?

**Comment by Qizot** [comment]
Ok, so I've came up with the following code to create the command (I know it is bad as it uses private API, but hey, it works)

```lua
            vim.api.nvim_create_user_command("OverseerRestartTask", function()
                local sidebar = require("overseer.task_list.sidebar").get()
                local task = sidebar._get_task_from_line(sidebar)
                if task then
                    task:restart()
                end
            end, {})
```

Would be nice if it was available by default.

**Comment by stevearc** [comment, marked as answer]
You can do `:OverseerQuickAction restart`
